README.md

```diff
@@ -30,24 +30,35 @@ Complete configuration for the different [node.js versions](https://github.com/n
 # https://github.com/nodejs/LTS
 sudo: false
 language: node_js
+os:
+ - linux
+before_install:
+  - 'case "${TRAVIS_NODE_VERSION}" in 0.*) export NPM_CONFIG_STRICT_SSL=false ;; esac'
+  - 'nvm install-latest-npm'
+install:
+  - 'if [ "${TRAVIS_NODE_VERSION}" = "0.6" ] || [ "${TRAVIS_NODE_VERSION}" = "0.9" ]; then nvm install --latest-npm 0.8 && npm install && nvm use "${TRAVIS_NODE_VERSION}"; else npm install; fi;'
 node_js:
-  - "0.8"   # end of life
-  - "0.10"  # end of life
-  - "0.12"  # end of life
-  - "4"     # maintenance lts
-  - "6"     # active lts
+  - "11"    # current
+  - "10"    # active lts
   - "8"     # active lts
-  - "9"     # current
+  - "6"     # active lts
+  - "4"     # end of life lts
+  - "0.12"  # end of life
+  - "0.10"  # end of life
+  - "0.8"   # end of life
+  - "0.6"   # end of life
 matrix:
   fast_finish: true
   allow_failures:
-    - node_js: "0.8"
-    - node_js: "0.10"
     - node_js: "0.12"
+    - node_js: "0.10"
+    - node_js: "0.8"
+    - node_js: "0.6"
 cache:
```

Comment:
these days, we can just do `cache: npm` or `cache: yarn` right?

Reply:
for `.npm` perhaps, i haven't tried and mildly prefer the explicit form; we'd need the list anyways for the nvm cache dir, unless travis adds support for some kind of `cache: nvm` thing